test/ut3_tester/core/test_suite_builder.pkb

```diff
@@ -767,6 +767,74 @@ create or replace package body test_suite_builder is
       );
   end;
 
+	procedure nested_contexts_2 is
+    l_actual      clob;
+    l_annotations ut3.ut_annotations;
+  begin
+    --Arrange
+    l_annotations := ut3.ut_annotations(
+      ut3.ut_annotation( 1, 'suite','Cool', null),
+      ut3.ut_annotation( 2, 'suitepath','path', null),
+      ut3.ut_annotation( 3, 'context','Level 1', null),
+      ut3.ut_annotation( 4,   'name','context_1', null),
```

Comment:
add some tests in parent context of nested context before/after nested context.
I wonder if this will still work as expected.

Reply:
Isn't that already done with `nested_contexts` test?

Reply:
Right